dags/openshift_nightlies/scripts/run_benchmark.sh

```diff
@@ -74,18 +70,17 @@ EOF
         exit 1
     fi
 }
+export UUID=$(uuidgen | head -c8)-$AIRFLOW_CTX_TASK_ID-$(date '+%Y%m%d')
+echo "############################################"
+echo "# Benchmark UUID: ${UUID}"
+echo "############################################"
 
 if [[ $PLATFORM == "baremetal" ]]; then
-    export UUID=$(uuidgen | head -c8)-$AIRFLOW_CTX_TASK_ID-$(date '+%Y%m%d')
     env >> /tmp/environment.txt
     run_baremetal_benchmark
-    echo $UUID
```

Comment:
@rsevilla87 thoughs on keeping lines 82 and 104? the bottom most part of the logs loads faster and having to search for the echo in line 75(which is def useful) might be a little more time consuming in tests like router where there's a lot of log output.